docs/source/conf.py

```diff
@@ -144,7 +144,7 @@
 intersphinx_mapping = {
     "python": ("https://docs.python.org/3/", None),
     "numpy": ("https://numpy.org/doc/stable/", None),
-    "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
+    "scipy": ("https://docs.scipy.org/doc/scipy-1.8.0/html-scipyorg/", None),
```

Comment:
Does it work with simply `'scipy': ('https://docs.scipy.org/doc/scipy', None),`? That seemed to work for numpy, and is the preferred link.

Reply:
+1 We also shouldn't pin the version of the docs here if in any way possible.

Reply:
Thanks for the feedback!  I changed it to `https://docs.scipy.org/doc/scipy` now. 

A few days ago pinning the 1.8 version was the only way to make it work for me, but it seems like now it is not necessary anymore. :)